src/analytics/store-analytics.js

```diff
@@ -10,12 +10,13 @@ const prestoClient = new PrestoClient({
   user: 'datalake-analytics-api'
 });
 
-const askAnalysis = (startDate, endDate, cnpjs, domain) => {
+const askAnalysis = (startDate, endDate, cnpjs, domains) => {
 	if (cnpjs.length > 500) {
 		throw 'Not allowed, you can query at most 500 cnpjs';
   }
 
 	const cnpjsInClause = cnpjs.map(c => `'${c}'`).join(",");
+	const domainInClause = domains.map(d => `'${d}'`).join(",");
```

Comment:
Please, fix this indentation. The whole function is crazily indented.

Reply:
Fxied

Reply:
Fxied